definitions/scenarios/upgrade_to_capsule_6_16.rb

```diff
@@ -80,6 +80,7 @@ class PostMigrations < Abstract
     def compose
       add_step(Procedures::RefreshFeatures)
       add_step(Procedures::Service::Start.new)
+      add_step(Procedures::Pulpcore::ContainerHandleImageData)
```

Comment:
I am thinking this should be the last step? One thing I talked to @ianballou about is making this step happen in a way that we communicate to users that the system is completely operable and usable while this is happening in case for some user it's a longer running task. The text output should also indicate this I think to help guide users.

Reply:
Indeed indeed, the migration should be triggered once the Satellite is back up and running after the upgrade. If there's an error, it shouldn't block the upgrade as well.
+1 to the text output too giving the user some info.

Reply:
@ehelms : Ack.. 👍🏼 
Having the step after `add_step(Procedures::Service::Start.new)` in the PostMigrations would get the services up but not return from the upgrade I guess..

Would adding it to PostMigrationCheck block the upgrade too?

Reply:
Services would be started where you have it now, but the firewall from maintenance mode would still be up (I know, not obvious from this). Take a look at https://github.com/theforeman/foreman_maintain/pull/763/files#diff-574e3a009c6f79e4ce0c1bbab07fc007071ea541bc8985ca4ffc4a6c7505d212L78-R113 where this lays out each individual item very clearly.

Reply:
Looking at that, feels like the right place would be in PostUpgradeChecks after post_upgrade checks step once the services are checked to be running etc.